### PR TITLE
Update faq.rst

### PR DESCRIPTION
**Pull request by JohnVeness**
Fixing some typos, and changes of things I didn't like as a matter of style. For discussion before committing.

### CODE REVIEW

**Comment by imsodin** [comment]
In my opinion the crossed-out SyncThing is a good way to make it very clear that this is incorrect, even without reading the sentence. Why do you want to remove this?

**Comment by JohnVeness** [comment]
It reads oddly to me. I've never seen text struck out in a book/magazine etc. when explaining what words not to use, for example.

File: users/faq.rst [hunk 1]
```diff
@@ -253,8 +253,8 @@ On Windows, escaping special characters is not supported as the ``\``
 character is used as a path separator. On the other hand, special characters
 such as ``[`` and ``?`` are not allowed in file names on Windows.
 
-Why is the setup more complicated than BTSync?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Why is the setup more complicated than BitTorrent Sync?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (AudriusButkevicius) [inline users/faq.rst]: Its now resilio sync I think.

File: users/faq.rst [hunk 2]
```diff
@@ -75,7 +75,7 @@ Is synchronization fast?
 
 Syncthing segments files into pieces, called blocks, to transfer data from one
 device to another. Therefore, multiple devices can share the synchronization
-load, in a similar way as the torrent protocol. The more devices you have online
+load, in a similar way to the torrent protocol. The more devices you have online
 (and synchronized), the faster an additional device will receive the data
```
Comment (AudriusButkevicius) [inline users/faq.rst]: The (and synchronized) bit can go too as its no longer relevant.

File: users/faq.rst [hunk 3]
```diff
@@ -344,8 +344,8 @@ My Syncthing database is corrupt
 
 This is almost always a result of bad RAM, storage device or other hardware. When the index database is found to be corrupt Syncthing cannot operate and will note this in the logs and exit. To overcome this delete the `database folder <https://docs.syncthing.net/users/config.html#description>`__ inside Syncthing's home directory and re-start Syncthing. It will then need to perform a full re-hashing of all shared folders. You should check your system in case the underlying cause is indeed faulty hardware which may put the system at risk of further data loss.
 
-I don't like the GUI / Theme. Can it be changed?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+I don't like the GUI or theme. Can it be changed?
```
Comment (AudriusButkevicius) [inline users/faq.rst]: Sholdn't it be the theme?

**Comment by AudriusButkevicius** [comment]
Yeah agree, the strike out shouldn't be there. If we want to empasize of what it's not supposed to be we can bold the not, oppose to make it harder to read.

File: users/faq.rst (hunk 2 above)
Comment (JohnVeness) [inline users/faq.rst]: Thanks, removed in my latest version.

File: users/faq.rst (hunk 1 above)
Comment (JohnVeness) [inline users/faq.rst]: Thanks, changed in my latest version.

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as 43aead5421310e50eb0f50f4051d0bb7ec6d73bc. Thanks, @JohnVeness!

**Comment by calmh** [comment]
@JohnVeness Thanks for doing all these doc updates. However, do you think you could use more descriptive commit subjects? The convention is "section/page: What changed", as in `users/contrib: Adding link to https://github.com/nhojb/SyncthingBar` or `users/config: Describe fsync`.

**Comment by JohnVeness** [comment]
Yes, can do. I considered that the diff would show what I changed, and the description would say why. But I can summarise the "what" in the description too.

**Pull request closed without merging**